Tests/Tests/STPPaymentIntentParamsTest.m

```diff
@@ -0,0 +1,66 @@
+//
+//  STPPaymentIntentParamsTest.m
+//  StripeiOS Tests
+//
+//  Created by Daniel Jackson on 7/5/18.
+//  Copyright © 2018 Stripe, Inc. All rights reserved.
+//
+
+#import <XCTest/XCTest.h>
+#import "STPPaymentIntentParams.h"
+
+@interface STPPaymentIntentParamsTest : XCTestCase
+
+@end
+
+@implementation STPPaymentIntentParamsTest
+
+- (void)testInit {
+    for (STPPaymentIntentParams *params in @[[[STPPaymentIntentParams alloc] initWithClientSecret:@"secret"],
+                                             [[STPPaymentIntentParams alloc] init],
+                                             [STPPaymentIntentParams new],
```

Comment:
no personal preference but mixing `alloc] init]` and `new]` right next to each other reads a bit weird :P

Reply:
Is this explicit to test that `new` actually calls `alloc] init]`? Seems like that's out of scope of something we need to test

Reply:
👍